man/randomA.Rd

```diff
@@ -0,0 +1,44 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/randomA.R
+\docType{methods}
+\name{randomA}
+\alias{randomA}
+\alias{randomA-numeric}
+\alias{randomA,numeric-method}
+\title{Generate random uniform interaction matrix}
+\usage{
+randomA(
+  n.species,
+  d = -0.5,
+  min.strength = -0.5,
+  max.strength = 0.5,
+  connectance = 0.02
```

Comment:
@antagomir BiocCheck() check gives notes for indentation of these four argument values. The problem with that  devtools::document()  automatically brings it back to these positions even if I change positions of them from R scripts. Any suggestion to fix this?

Reply:
Does it work to remove the newline? d= -0.5, min.strength = -0.5, max.strength = 0.5, connectance = 0.02

Reply:
Checks for indentation do not really work in the case of function arguments and can be ignored. I never found a way to deal with this. Just ignore the warning if this in function arguments.